Explorer/src/components/containerTable.tsx

```diff
@@ -88,10 +86,10 @@ export default function ContainerTable({
             <span>
               <Dropdown inline label="STATUS" dismissOnClick={true}>
```

Comment:
`dismissOnClick={true}` only works for the first click. I don't have a solution for this one